README.md

```diff
@@ -9,7 +9,7 @@ There are some tests that all extensions should run to ensure that overlayed wil
 
 ## How
 
-`npm i @overlayed-app/common-ext-tests`
+`npm i @overlayed-app/common-ext-tests -D`
```

Comment:
i'd prefer to put the `-D` after the `i` so that it is logically grouped as a flag to the install operation - I think most users would expect to see it there instead of at the end.